Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/Snippet.md

```diff
@@ -0,0 +1,26 @@
+# ``docc/Snippet``
```

Comment:
This page won't be displayed as long as the snippet symbol isn't in the symbol graph file of directives that this catalog uses.

To include it you'd need to remove [this `hiddenFromDocumentation = true` line](https://github.com/swiftlang/swift-docc/blob/main/Sources/SwiftDocC/Semantics/Snippets/Snippet.swift#L33) and then run `swift run generate-symbol-graph` in the DocC repo and commit the modified symbol graph file.

Reply:
Thank you! I don't know how long it would have taken me to hunt down that detail without your guidance!